modules/roles_profiles/manifests/profiles/cltbld_user.pp

```diff
@@ -83,7 +83,7 @@
         # require => User['cltbld'],
       }
 
-      $sudo_commands = ['/sbin/reboot']
+      $sudo_commands = ['/sbin/reboot', '/usr/sbin/softwareupdate']
```

Comment:
We don't want to allow this command, but rather the python script I wrote that only allows updating 'safari' packages.

Reply:
Oh got it. For some reason I thought that would go in the test itself. Will fix